.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
```

Comment:
Will this do anything?
Given we pin versions using hashes, either it won't do anything or it will spam with PR updates using vx.y.z versioning I suppose? Or does it support bumping up using hashes as well??